### DISCUSSION TITLE
Feature Request: folder/subfolder structure of webradios

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#587] [Fixed Issues]
Current state:
I have more than 100 webradios configured. Browsing the list has become confusing.

Desired state:
I want to organize my webradios in the library view according to my criteria in folders/subfolders.

**Comment by R101G** [comment]
+1 for this. I would also like to be able to bookmark radio feeds in the same way as I can albums.

**Comment by kan-ibal** [comment]
I keep my radio stream links as m3u playlists in /var/lib/mpd/playlists. This way I can easily create and maintain them in Cantata(a lot of radio browsers) and play via every mpd client.
rAudio creates "strange" playlist format in /srv/http/data/playlists, what about unifying this and symlink rAudio playlist folder to /var/lib/mpd/playlists and change rAudio playlist format to m3u to use the same playlists as mpd?
